com.unity.netcode.gameobjects/Samples~/Bootstrap/Scripts/BootstrapManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Unity.Netcode;
+using UnityEngine;
+
+namespace BoostrapSample
```

Comment:
(minor)
should we use a different namespace? I'm thinking `Unity.Netcode` variants, something like `Unity.Netcode.Samples` etc.

Reply:
I went with your suggestion here, `Unity.Netcode.Samples`.